Hestia/app/src/main/java/hestia/backend/exceptions/ServerExceptions/UndefinedServerException.java

```diff
@@ -0,0 +1,7 @@
+package hestia.backend.exceptions.ServerExceptions;
+public class UndefinedServerException extends ServerException {
```

Comment:
Also I think its strange that a DefinedServerException extends Exception but an UndefinedServerException extends ServerException. This asymmetry is not desirable.